src/etc/inc/dyndns.class

```diff
@@ -507,6 +511,29 @@
 					curl_setopt($ch, CURLOPT_USERPWD, $this->_dnsUser.':'.$this->_dnsPass);
 					curl_setopt($ch, CURLOPT_POSTFIELDS, $post_data);
 					break;
+				case 'noip-v6':
+				case 'noip-free-v6':
+					$needsIP = TRUE;
+					$server = "https://dynupdate.no-ip.com/ducupdate.php";
+					$port = "";
+					if ($this->_dnsServer) {
+						$server = $this->_dnsServer;
+					}
+					if ($this->_dnsPort) {
+						$port = ":" . $this->_dnsPort;
+					}
+					if (($this->_dnsService == "noip-free-v6") &&
+					    ($this->_forceUpdateNeeded == true) &&
+					    ($this->_dnsDummyUpdateDone == false)) {
+						// Update the IP to a dummy value to force No-IP free accounts to see a change.
+						$iptoset = "fd00:d::1";
+						$this->_dnsDummyUpdateDone = true;
+						log_error(sprintf(gettext('Dynamic DNS %1$s (%2$s): Processing dummy update on No-IP free account. IP temporarily set to %3$s'), $this->_dnsService, $this->_dnsHost, $iptoset));
```

Comment:
Please break this long line improve readability

Reply:
Adopted suggestion: broke the line.